docs/source/query-plans.mdx

```diff
@@ -444,6 +446,16 @@ You can view the query plan for a particular operation in any of the following w
     - Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 - As direct output from the `@apollo/gateway` library (see below)
 
+### Defer node
+A `Defer` node corresponds to one or more `@defer` at the same level of "nestedness" in the planned query.
+
+It contains a "primary block" and an array of "deferred blocks". The "primary block represents the part of the query that is not deferred (so the part of the query up until we reach the @defer(s) this handle), while each "deferred block" correspond to the deferred part of one of the `@defer` handle by the node.
+
+Read more about how `@defer` operatates at [Apollo Router support for @defer](/router/executing-operations/defer-support/).
+
+### Condition node
+A `Condition` node is used to split a plan into an if-else branch. This is commonly used for `@skip` and `@include` to have a query plan that can select different nodes based on the provided runtime variables.
+
```

Comment:
It looks like these need to move up to the previous H2, right under the `### Flatten node` section, so they are not part of `## Viewing query plans`

Additionally, could you please add examples of what the node looks like, as is done in the other node sections? E.g.  for [Flatten node](https://www.apollographql.com/docs/federation/query-plans/#flatten-node)